content/master/guides/change-logs.md

```diff
@@ -0,0 +1,271 @@
+kind: DeploymentRuntimeConfig
+metadata:
+  name: enable-changelogs
+spec:
+  deploymentTemplate:
+    spec:
+      selector: {}
+      template:
+        spec:
+          containers:
+          - name: package-runtime
+            args:
+            - --enable-changelogs
+            volumeMounts:
+            - name: changelogs-vol
+              mountPath: /var/run/changelogs
+          - name: changelogs-sidecar
+            image: xpkg.crossplane.io/crossplane/changelogs-sidecar:v0.0.1
+            volumeMounts:
+            - name: changelogs-vol
+              mountPath: /var/run/changelogs
+          volumes:
+          - name: changelogs-vol
+            emptyDir: {}
+  serviceAccountTemplate:
+    metadata:
+      name: provider-kubernetes
+EOF
+```
+
+### Install the provider
+
+Install the {{<hover label="provider" line="7">}}provider{{</hover>}} and
+instruct it to use the {{<hover label="provider" line="8">}}DeploymentRuntimeConfig{{</hover>}}
+that was just created.
+
+```yaml {label="provider",copy-lines="all"}
+cat <<EOF | kubectl apply -f -
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-kubernetes
+spec:
+  package: xpkg.crossplane.io/crossplane-contrib/provider-kubernetes:v0.18.0
+  runtimeConfigRef:
+    apiVersion: pkg.crossplane.io/v1beta1
+    kind: DeploymentRuntimeConfig
+    name: enable-changelogs
+EOF
+```
+
+### Configure permissions
+
+In order for the provider to create Kubernetes resources within the control
+plane, it must be granted the appropriate permissions. This guide only creates a
+`ConfigMap`, so only permissions for that resource type are needed.
+
+{{<hint "important">}} This guide grants specific permissions to the provider
+for example purposes. This approach isn't intended to be representative of a
+production environment. More examples on configuring `provider-kubernetes` can
+be found in its [examples directory](https://github.com/crossplane-contrib/provider-kubernetes/tree/main/examples/provider).
+{{</hint>}}
+
+```yaml {label="rbac",copy-lines="all"}
+cat <<EOF | kubectl apply -f -
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: configmap-edit
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - "*"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: provider-kubernetes-configmap-edit
+subjects:
+  - kind: ServiceAccount
+    name: provider-kubernetes
+    namespace: crossplane-system
+roleRef:
+  kind: ClusterRole
+  name: configmap-edit
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: kubernetes.crossplane.io/v1alpha1
+kind: ProviderConfig
+metadata:
+  name: default
+spec:
+  credentials:
+    source: InjectedIdentity
+EOF
+```
+
+### Create a resource
+
+Now that the provider is installed and configured with change logs enabled,
+create a resource that will generates change logs entries reflecting the actions
```

Comment:
```suggestion
create a resource that will generate change logs entries reflecting the actions
```

Reply:
thanks @jeanduplessis for the review! i fixed this grammar issue 🙇‍♂️